### DISCUSSION TITLE
[Unstable] Bots pick plant products and pet items and keep those in their inventory.

### DISCUSSION THREAD

**Discussion by AzureNocte** [FakeFishGames/Barotrauma#13733] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In current version, bots will pick plant products and items drop by pets and put them into steel cabinets, with or without cleanup items order. But in 1.4.1.0, I noticed that only bots with cleanup items order will act like before, other bots just pick items and keep those in their inventory and gradually get full loaded. 
This causes a problem, bots would tend to drop items(like oxygen tank or helmet) in emergency situation, and leave those everywhere.

### Reproduction steps

1. Start campaign and have some bots
2. get several plants and pets
3. give no order to bots
4. observe their behavior, check their inventory

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

Other

### -

v1.4.1.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I [created a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13830), we'll look into it!

**Comment by NilanthAnimosus** [comment]
This has been fixed in our private repo now and should be included in an upcoming hotfix!
